gammapy/cube/tests/test_counts.py

```diff
@@ -82,3 +84,21 @@ def test_fill_map_counts_keyerror(events):
     cntmap = WcsNDMap.create(binsz=0.1, npix=10, axes=[axis])
     with pytest.raises(KeyError):
         fill_map_counts(cntmap, events)
+
+@requires_data('gammapy-extra')
+def test_fill_map_counts_multiple_energy_axes(events):
+    axis_mc = MapAxis([0, 2, 4], name='energy_mc', unit='TeV')
+    axis_reco = MapAxis([0.001, 1000.], name='energy_reco', unit='TeV')
+    axis = MapAxis([0.001, 1000.], name='energy', unit='TeV')
+
+    cntmap1 = WcsNDMap.create(binsz=1, npix=10, coordsys='GAL', axes=[axis, axis_mc])
```

Comment:
Suggest to remove `coordsys="GAL" and "binsz=1".
Doesn't add any value to the test, so less code to read is better, no?
I guess the `npix` you could keep if you want to be a tiny bit faster / more memory efficient in the test.

Reply:
It simply allows to have non zeros entries in the final map and help asserting the correct values